lldp/tx.c

```diff
@@ -330,8 +330,19 @@ void process_tx_idle(UNUSED struct lldp_agent *agent)
 	return;
 }
 
-void process_tx_shutdown_frame(struct port *port, struct lldp_agent *agent)
+/* we ignore 'state' value in the case that we have recently transitioned
+ * to the shutdown state (in the case of the 'tx' state change) to allow
+ * for transmitting the ttl==0 as required by the IEEE standard. */
+void process_tx_shutdown_frame(struct port *port, struct lldp_agent *agent,
+				bool ignoreState)
```

Comment:
A nit: you call it ignoreState, but you are ignoring status. It would be nice if the sense of the two linked things were the same (speaking as one that doesn't know the code well).

Reply:
Okay - if it's good with that nit corrected, I can fix and commit with your Acked-by if you give it.